src/fragments/auth.channelPermissions.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from "@apollo/client";
+
+export const channelFragment = gql`
+  fragment ChannelFragment on Channel {
+    id
+    isActive
+    name
+    slug
+    currencyCode
+    defaultCountry {
+      code
+      country
+    }
+    stockSettings {
+      allocationStrategy
+    }
+  }
+`;
+
+export const fragmentUserWithChannels = gql`
+  fragment UserWithChannels on User {
+    id
+    email
+    firstName
+    lastName
+    isStaff
+    userPermissions {
+      ...UserPermission
+    }
+    avatar {
+      url
+    }
```

Comment:
**suggestion**: Avatar size defaults to 4096. Do we need so large? It's probably good idea to specify something smaller, like 128.
```suggestion
    avatar(size: 128) {
      url
    }
```